core/domain/app_feedback_report_validators.py

```diff
@@ -61,86 +68,81 @@ def _get_external_id_relationships(cls, item):
                         item.id, item.web_report_info_schema_version,
+                        feconf.MINIMUM_WEB_REPORT_SCHEMA_VERSION,
                         feconf.CURRENT_WEB_REPORT_SCHEMA_VERSION))
-            elif item.web_report_info_schema_version < (
-                    feconf.MINIMUM_WEB_REPORT_SCHEMA_VERSION):
-                cls._add_error(
-                    'report schema %s' % (
-                        base_model_validators.ERROR_CATEGORY_VERSION_CHECK),
-                    'Entity id %s: web report schema version %s is less '
-                    'than the minimum version %s' % (
-                        item.id, item.web_report_info_schema_version,
-                        feconf.MINIMUM_WEB_REPORT_SCHEMA_VERSION))
 
     @classmethod
-    def _validate_submitted_on_datetime(cls, item):
-        """Validate that submitted_on of model is less than current time and
-        greater than the earliest possible date of submissions (no earlier than
-        March 2021).
+    def _validate_created_on_datetime(cls, item):
+        """Validates that the created_on date of the model is less than the
+        current time and greater than the earliest possible date of submissions
+        (no earlier than March 2021).
 
         Args:
             item: datastore_services.Model. AppFeedbackReportModel to validate.
         """
         current_datetime = datetime.datetime.utcnow()
-        if item.submitted_on > current_datetime:
+        if item.created_on > current_datetime:
             cls._add_error(
-                'submitted_on %s' % (
+                'created_on %s' % (
                     base_model_validators.ERROR_CATEGORY_DATETIME_CHECK),
-                'Entity id %s: The submitted_on field has a value %s which is '
+                'Entity id %s: The created_on field has a value %s which is '
                 'greater than the time when the job was run' % (
-                    item.id, item.submitted_on))
-        if item.submitted_on < EARLIEST_VALID_DATETIME:
+                    item.id, item.created_on))
+        if item.created_on < EARLIEST_VALID_DATETIME:
             cls._add_error(
-                'submitted_on %s' % (
+                'created_on %s' % (
                     base_model_validators.ERROR_CATEGORY_DATETIME_CHECK),
-                'Entity id %s: The submitted_on field has a value %s which is '
+                'Entity id %s: The created_on field has a value %s which is '
                 'less than the earliest possible submission date' % (
-                    item.id, item.submitted_on))
+                    item.id, item.created_on))
 
     @classmethod
     def _validate_expired_reports_are_scrubbed(cls, item):
-        """Validate that if the submitted_on of model is less than 90 days
-        before the current date, then the scrubbed_by field is non-None.
+        """Validates that if the submitted_on of model is less than 92 days
```

Comment:
Could we avoid using exact numbers like '92' here & elsewhere for values that are configurable? Consider instead referencing the constant if readers want to get an up-to-date snapshot of state.

Reply:
Done